src/extension.ts

```diff
@@ -48,9 +57,12 @@ export async function activate(context: vscode.ExtensionContext) {
         initializeManifestPaths(context)
 
         const toolkitSettings = new DefaultSettingsConfiguration(extensionSettingsPrefix)
+
+        const endpointsProvider = makeEndpointsProvider()
```

Comment:
Do we need the const? Can we just call this when creating the regionProvider?

Reply:
Nested constructors don't read nicely (I initially wrote it like that and changed it to this)